elections/2023/governance-committee-election.md

```diff
@@ -0,0 +1,108 @@
+# OpenTelemetry 2023 Governance Committee election
+
+Please see **[the list of candidates running in the election here](./governance-committee-candidates.md)**.
+
+This document lays out the 2023 election process per the OpenTelemetry governance committee [charter document](../../governance-charter.md#establishment-of-a-governance-committee) requirements.
+
+This election should fill four seats.
+
+Election schedule:
+
+* 13 October 2023 - final deadline to submit nominations
+* 16 October 2023 - official nominees list published
+* 23 October 2023 to 25 October 2023 - voting takes place
+* 27 October 2023 - results announced
+
+We highly encourage participation in this election cycle to ensure that the community is well-represented by the Governance Committee.
+
+# TL;DR
+
+* If you've been nominated or are willing to nominate yourself: check the [charter document](../../governance-charter.md) and confirm you are ready for the commitment. Make sure to provide all necessary information before 13 October 2023 00:00 UTC
+* If you are an active community member: confirm that you are on the voters list (see [election announcements issue](https://github.com/open-telemetry/community/issues/1561)) or register yourself before 20 October 2023 23:59 UTC.
+* Vote between 23 October 2023 00:00 UTC and 25 October 2023 23:59 UTC via the [voting link](https://vote.heliosvoting.org/helios/elections/1ee70ee4-11ce-11ee-aaf8-0a8c9aac83f9/view)
+* Keep being awesome and contributing to the project!
+
+# Vacancies
+
+Current governance committee members (alphabetical order):
+
+* [Alolita Sharma](https://github.com/alolita), Apple, until October 2024
+* [Ben Sigelman](https://github.com/bhs), Lightstep, until October 2023
+* [Bogdan Drutu](https://github.com/BogdanDrutu), Splunk, until October 2023
+* [Daniel Dyla](https://github.com/dyladan), Dynatrace, until October 2024
+* [Juraci Paixão Kröhling](https://github.com/jpkrohling), Grafana Labs, until October 2023
+* [Morgan McLean](https://github.com/mtwo), Splunk, until October 2024
+* [Ted Young](https://github.com/tedsuo), Lightstep, until October 2023
+* [Trask Stalnaker](https://github.com/trask), Microsoft, until October 2024
+* [Yuri Shkuro](https://github.com/yurishkuro), Facebook, until October 2023
+
+Five people must be elected in this election, each with two-year terms. Note that five current Governance Committee members end terms as part of this election cycle, though they are welcome to run again for one of the five vacancies.
+
+After this election and until the next election, the Governance Committee will have nine seats, all of which will be elected positions. Please refer to the [charter document](../../governance-charter.md#establishment-of-a-governance-committee) for more information.
```

Comment:
It's time to remove this paragraph. It's from a time when we were changing the size of the GC.